### ISSUE TITLE
Request for Clarification on SatMAE++ Accuracy in EuroSAT Benchmark

### ISSUE THREAD

**Issue by kzigzag** [techmn/satmae_pp#8]
Hello!
Thank you for sharing the code and the impressive results.

<img width="1107" height="529" alt="Image" src="https://github.com/user-attachments/assets/98341f7f-ed14-4a34-8e67-ad91f2115a64" />

It is noteworthy that ‘SatMAE++ surpasses the performance score of the state-of-the-art SatMAE without using the multi-spectral information and achieves an accuracy score of 99.01%.' 
However, this is not consistent with the 99.04% of EuroSAT results in Table 5.
Could you please confirm the actual experimental accuracy?

**Comment by kzigzag** [comment]
By the way, the link for the dataset split txt file could not be accessed by the https as follows.
This has resulted in a difference between the results we achieved when reproducing the fine-tuning on our own data split and the results presented in the paper.

https_list =
{
EuroSAT: 
https://storage.googleapis.com/remote_sensing_representations/eurosat-train.txt
https://storage.googleapis.com/remote_sensing_representations/eurosat-val.txt
https://storage.googleapis.com/remote_sensing_representations/eurosat-test.txt

UC Merced: 
https://storage.googleapis.com/remote_sensing_representations/uc_merced-train.txt
https://storage.googleapis.com/remote_sensing_representations/uc_merced-val.txt
https://storage.googleapis.com/remote_sensing_representations/uc_merced-test.txt

BigEarthNet: 
https://storage.googleapis.com/remote_sensing_representations/bigearthnet-train.txt
https://storage.googleapis.com/remote_sensing_representations/bigearthnet-val.txt
https://storage.googleapis.com/remote_sensing_representations/bigearthnet-test.txt
}

I wonder if you could share them. Thanks so much.

**Comment by techmn** [comment]
> Hello! Thank you for sharing the code and the impressive results.
> 
> <img alt="Image" width="1107" height="529" src="https://private-user-images.githubusercontent.com/115205786/494711666-98341f7f-ed14-4a34-8e67-ad91f2115a64.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTkyMzcxNTYsIm5iZiI6MTc1OTIzNjg1NiwicGF0aCI6Ii8xMTUyMDU3ODYvNDk0NzExNjY2LTk4MzQxZjdmLWVkMTQtNGEzNC04ZTY3LWFkOTFmMjExNWE2NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkzMFQxMjU0MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZGZlOGQxMWM1ZWJjZWM4YmNjMDVmNTQwMTI4NmVjYjhiNzQ4ZGZjYTRkMWUxMzMzZWJlMWE4YmIyOTMyN2E4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.kDH5TCu1PbadmNaEqVUgQYm5GaKJzi2ta9lSJjyhOKs">
> It is noteworthy that ‘SatMAE++ surpasses the performance score of the state-of-the-art SatMAE without using the multi-spectral information and achieves an accuracy score of 99.01%.' However, this is not consistent with the 99.04% of EuroSAT results in Table 5. Could you please confirm the actual experimental accuracy?

Hi,

Thank you for pointing out the issue. **Actual score is** what is mentioned in the table as **99.04**. Other is the typo.

**Comment by techmn** [comment]
> By the way, the link for the dataset split txt file could not be accessed by the https as follows. This has resulted in a difference between the results we achieved when reproducing the fine-tuning on our own data split and the results presented in the paper.
> 
> https_list = { EuroSAT: https://storage.googleapis.com/remote_sensing_representations/eurosat-train.txt https://storage.googleapis.com/remote_sensing_representations/eurosat-val.txt https://storage.googleapis.com/remote_sensing_representations/eurosat-test.txt
> 
> UC Merced: https://storage.googleapis.com/remote_sensing_representations/uc_merced-train.txt https://storage.googleapis.com/remote_sensing_representations/uc_merced-val.txt https://storage.googleapis.com/remote_sensing_representations/uc_merced-test.txt
> 
> BigEarthNet: https://storage.googleapis.com/remote_sensing_representations/bigearthnet-train.txt https://storage.googleapis.com/remote_sensing_representations/bigearthnet-val.txt https://storage.googleapis.com/remote_sensing_representations/bigearthnet-test.txt }
> 
> I wonder if you could share them. Thanks so much.

-------------------------------------------------------------------------
You can find the split files [here](https://github.com/techmn/satmae_pp/tree/main/fmow_rgb_data_split). I only have the files for Eurosat.
